modules/datastore/tests/src/Unit/Storage/DatabaseTableTest.php

```diff
@@ -67,6 +69,17 @@ public function testGetSchema() {
           "mysql_type" => "text",
         ],
       ],
+      "indexes" => [
+        "idx1" => [
+          "first_name",
+        ],
+      ],
+      "fulltext indexes" => [
+        "ftx1" => [
+          "first_name",
+          "last_name",
+        ],
+      ],
```

Comment:
Could we have one "indexes" array, similar to what we're planning to do for data-dictionaries?
e.g.
```
"indexes" => [
  "index_name" => {
    "fields" => [
      "first_name",
      "last_name",
    ],
    "type" => "fulltext" or "btree" or "spatial",
  },
]
```

Reply:
That is what I think we should do if we switch to Frictionless' table schema for the schema object. For better or worse we are still using the [Drupal schema API](https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Database!database.api.php/group/schemaapi/9.0.x) here.

Reply:
I'm fine with just sticking to the Drupal schema for now - may be worth revisiting later though